### ISSUE TITLE
FileUploader: Value State Message is not being announced by the Screen Reader

### ISSUE THREAD

**Issue by achuth2912** [UI5/webcomponents#13549]
### Describe the bug

When we upload a file using the file uploader component the valueStateMessage is not being announced by the File Uploader component.

### Isolated Example

https://stackblitz.com/edit/github-xrgckykb?file=src%2FApp.tsx

### Reproduction steps

1. Upload a File
2. Screen reader does not announce the valueStateMessage
3.
...

### Expected Behaviour

Screen reader must announce the valueStateMessage on upload.

### Screenshots or Videos

<img width="1728" height="955" alt="Image" src="https://github.com/user-attachments/assets/921ad660-4a2e-4e76-8dc1-bcf3f7215250" />

### UI5 Web Components for React Version

2.19.2

### UI5 Web Components Version

2.19.2

### Browser

Chrome

### Operating System

Windows, Mac

### Relevant log output

```shell

```

### Organization

SAP

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.

**Comment by IlianaB** [comment]
Hello @UI5/ui5-team-ctr-bal ,
please have a look at this accessibility issue, reproducible with the stackblitz sample above with 2.19.2 version and also [here](https://ui5.github.io/webcomponents/play/#eNq9U1Fv2jAQ_ivXPLUIEzpp0sRCNdGOp3Wt1PIwiRdjH8SdY0exE4oQ_31nJ0Ggbt2epqAEf_fdd-fv7H2ijMTXUe4LnUz2ieEFJpNTcJgIazwaT3B2wRiUmu82la2NZGurJTB2szTZxd3D7fOPx68QkgIQvqC52UyXCZplQlj7ZDlySSugX-b8TmNYDCbG-suJxLUyKK9gT5hULhSbgLEGPxNwCPmxQoyvbSVQMmG1rRiXL7XzZ1zI0l7__5fLCvQcRM4rh548WDzP2adoQxcKVhPeKNyWtvLLBDqjCdwq6fOpxEYJZHExBGWUV1wzJ7jG6fVo3Il55anmEy9KjVnarsjltLOZ_q6s3EFsjaRXXPzsxhd3MoGGV5eMOV7OjqHbELnqCvxm6AxNN3jIpGqO4jmqTU6uXI_H5WuXXquPTPMV6mAgcZwyG41srehVl9pyiRVt3uV2GzsyN4uIAoeWCoGapUedo-q5hAElSV5g5U90CeZCYBlcHZVyHZGG6xqZ89yHpr_jhnvVYNdv3I-2ISHyngLtHp3jm0B5MHoHj3fz2JUDcgQr-HA_A14hcK3tFuUoS0klqqVvGm3hPv7HG0WHVVSq9OB3ZeiysLLWGIyqRFhyZUYvjhqiUxeJcehh1O3Q0_4ivju-5DDspU6uf4-c3v10cKqSK4kwSJdGFeHwApmbnoRdXQY4Nki3421u7OAs_wsZlW5xJSwhhmo68sj5dE7eLTrrer2_Js1q7635Z_q3cKx69rt8psiRLqsdaJuWHA6_AK6A2S0) with 2.22.0

**Actual result:**
valueStateMessage is not read out by screen reader (reported with NVDA. I am testing with JAWS 2026.2605.43 and observe the same result)

**Expected result:**
Similar to ui5-input - value state message should be read out by screen readers.

Regards,
Iliana
